docs/src/manual/models.md

```diff
@@ -241,6 +241,18 @@ with optimizer MOIB.LazyBridgeOptimizer{GLPK.Optimizer}
 The backend is a `MOIU.CachingOptimizer` in the state `EMPTY_OPTIMIZER` and mode
 `AUTOMATIC`.
 
+Alternatively, use [`unsafe_backend`](@ref) to access the `GLPK.Optimizer`
+object at the bottom of the stack.
```

Comment:
I would consider using the word innermost here, since its used in other descriptions.